docs/Azure_IoT_Central_integration/Concepts/LwM2M_mappings.md

```diff
@@ -0,0 +1,61 @@
+# LwM2M mappings
```

Comment:
Perhaps Concepts docs should be the same for hub and central? 
Or if you want to keep it in this shape, we should add more details here - in IoT Central many of described mechanisms are named differently -> for example Device Twins -> Properties, Device-to-cloud -> Telemetry, Direct Methods -> Commands. I think you should check the example application and spend some time with the GUI to gain some experience with the naming conventions.

Reply:
let's go for a shared concepts section for Hub and Central as for now